CHANGELOG.md

```diff
@@ -1,3 +1,91 @@
+- [Support `blendMode` parameter when drawing text on Canvas](https://github.com/JetBrains/compose-multiplatform-core/pull/470)
+- [Support compositing strategy on `graphicsLayer`](https://github.com/JetBrains/compose-multiplatform-core/pull/486)
+- [Add `minLines` parameter to `BasicTextField`](https://github.com/JetBrains/compose-multiplatform-core/pull/469)
+- [Support `painterResource` function common source set](https://github.com/JetBrains/compose-multiplatform/pull/2793)
+
+### Fixes
+
+- [Fix performing `ComposeScene.sendKeyEvent` an unintentional non-local return](https://github.com/JetBrains/compose-multiplatform-core/pull/398)
+- [Actually remember provider in `rememberComponentRectPositionProvider`](https://github.com/JetBrains/compose-multiplatform-core/pull/444)
+- [Fix merging multiple selection areas inside `SelectionContainer`](https://github.com/JetBrains/compose-multiplatform/issues/2899)
+- [Fix crash on selecting text if layout result is invalid](https://github.com/JetBrains/compose-multiplatform-core/pull/482)
+- [Fix crash `SplitPane` with a modifier](https://github.com/JetBrains/compose-multiplatform/issues/2214)
+
+## iOS
+
+[iOS support is in Alpha now](README.md#ios)
+
+## Desktop
+
+### Features
+
+- [Support external drag and drop](https://github.com/JetBrains/compose-multiplatform-core/pull/391)
+- [Animate scroll initiated by mouse wheel input](https://github.com/JetBrains/compose-multiplatform-core/pull/438)
+- [Remove AWT dependency in `rememberCursorPositionProvider`](https://github.com/JetBrains/compose-multiplatform-core/pull/432)
+
+### Fixes
+
+- [Fix showing a surface behind `AlertDialog` shape](https://github.com/JetBrains/compose-multiplatform/issues/1269)
+- [Fix `AlertDialog` shadow size](https://github.com/JetBrains/compose-multiplatform/issues/2857)
+- [Fix `AlertDialog` width to match the maximum of its content](https://github.com/JetBrains/compose-multiplatform/issues/2836)
+- [Fix `ContextMenuArea` to put the context menu representation in the same box with the content](https://github.com/JetBrains/compose-multiplatform/issues/2729)
+- [Replace using global mouse position with known position for context menus, dropdown menus and tooltips](https://github.com/JetBrains/compose-multiplatform-core/pull/437)
+- [Limit error dialog width in case of long error message](https://github.com/JetBrains/compose-multiplatform-core/pull/441)
+- [Fix window flashing background with unspecified size](https://github.com/JetBrains/compose-multiplatform-core/pull/442)
+- [Unregister window/dialog listeners when disposing](https://github.com/JetBrains/compose-multiplatform-core/pull/446)
+- [Fix applying `ComposePanel` visibility](https://github.com/JetBrains/compose-multiplatform/issues/2656)
+
+## Web
+
+- [`compose.web` is renamed to `compose.html`](https://github.com/JetBrains/compose-multiplatform/pull/2997)
```

Comment:
Also, we should add:
- [The new experimental `Compose for Web` based on Kotlin/Wasm is available](README.md#web)